bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/door-status-channel-types.xml

```diff
@@ -6,39 +6,63 @@
 	<channel-type id="driver-front-channel">
 		<item-type>String</item-type>
 		<label>Driver Door</label>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="driver-rear-channel">
 		<item-type>String</item-type>
 		<label>Driver Door Rear</label>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="passenger-front-channel">
 		<item-type>String</item-type>
 		<label>Passenger Door</label>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="passenger-rear-channel">
 		<item-type>String</item-type>
 		<label>Passenger Door Rear</label>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="hood-channel">
 		<item-type>String</item-type>
 		<label>Hood</label>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="trunk-channel">
 		<item-type>String</item-type>
 		<label>Trunk</label>
+		<tags>
+			<tag>Status</tag>
+			<tag>OpenState</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="window-driver-front-channel">
 		<item-type>String</item-type>
 		<label>Driver Window</label>
 		<tags>
 			<tag>Status</tag>
-			<tag>OpenState</tag>
+			<tag>OpenLevel</tag>
```

Comment:
Why change this? AFAICT this is a boolean channel represented as String. open/closed and not 34% or something, so OpenState would be more suitable? 
Same for the others in this file where `OpenState` is replaced by `OpenLevel`

Reply:
But I guess the better alternative is to tag them as OpenState (even if they can have more than two states). So I will revert this and adjust Merc to match. (Also Volvo, Tesla and Renault).